GVFS/GVFS.Mount/InProcessMount.cs

```diff
@@ -44,6 +45,8 @@ public class InProcessMount
         private HeartbeatThread heartbeat;
         private ManualResetEvent unmountEvent;
 
+        private readonly Dictionary<string, string> treesWithRecentlyDownloadedCommits = new Dictionary<string,string>();

```

Comment:
This dictionary lives for the lifetime of the process? We say "recently downloaded" but also we don't clear this on some schedule. Are we concerned about memory bloat over time?

Reply:
Updated the variable name to just "treesWithDownloadedCommits."
After the changes to skip this behavior if prefetch has occurred I am not concerned about memory cost of this over time.